packages/services/f-metadata/README.md

```diff
@@ -91,43 +82,42 @@ brazePromise.then(brazeDispatcher => {
 
 ### Config Object
 
-
-#### `config.apiKey`
+`config.apiKey`
```

Comment:
Worth keeping these titles for deep-linking

Reply:
Yes.  I've done that in the next PR.